package.json

```diff
@@ -41,13 +41,13 @@
   },
   "scripts": {
     "start": "API_URL=${API_URL:=http://localhost:8000} USE_AXE=${USE_AXE:=true} ./scripts/env.sh && cp env-config.js ./public/ && BROWSER=none react-scripts start",
-    "start-with-test-api": "API_URL=https://hauki-test.oc.hel.ninja USE_AXE=${USE_AXE:=true} ./scripts/env.sh && cp env-config.js ./public/ && react-scripts start",
+    "start-with-test-api": "API_URL=https://hauki-api-test.agw.arodevtest.hel.fi USE_AXE=${USE_AXE:=true} ./scripts/env.sh && cp env-config.js ./public/ && react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
     "test-cypress": "start-server-and-test start http://localhost:3000 cypress-run-chrome",
     "test-cypress-headless": "start-server-and-test start http://localhost:3000 cypress-run-chrome-headless",
-    "test-cypress-test-env": "CYPRESS_BASE_URL=https://hauki-admin-ui.dev.hel.ninja yarn cypress-run-chrome-headless",
-    "test-cypress-with-test-api": "API_URL=https://hauki-test.oc.hel.ninja start-server-and-test start http://localhost:3000 cypress-run-chrome-headless",
+    "test-cypress-test-env": "CYPRESS_BASE_URL=https://hauki-admin-ui-test.agw.arodevtest.hel.fi yarn cypress-run-chrome-headless",
+    "test-cypress-with-test-api": "API_URL=https://hauki-api-test.agw.arodevtest.hel.fi start http://localhost:3000 cypress-run-chrome-headless",
```

Comment:
It seems that this line is missing the 'start-server-and-test' command in front of the start 🤔